File: templates/Uwp/_composition/Prism/Feature.UnitTests.Core.SampleData.MasterDetail.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
﻿//{[{
using System.Linq;
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public async Task EnsureSampleDataServiceReturnsModelDataAsync()
        {
            var dataService = new SampleDataService();
            var actual = await dataService.GetSampleModelDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}

File: templates/Uwp/_composition/_shared/Feature.UnitTests.Core.SampleData.ImageGallery.xUnit._VB/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.vb
﻿'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Fact>
    Public Sub EnsureSampleDataServiceReturnsGalleryData()
        Dim actual = SampleDataService.GetGallerySampleData()

        Assert.NotEmpty(actual)
    End Sub
    '}]}
End Class
